exercises/handling-errors/README.md

```diff
@@ -0,0 +1,63 @@
+## Exercise #1: Handling Errors
+
+During this exercise, you will:
+
+- Throw and handle exceptions in Temporal Workflows and Activities
+- Use non-retryable errors to fail an Activity
+- Locate the details of a failure in Temporal Workflows and Activities in the Event History
+
+Make your changes to the code in the `practice` subdirectory (look for `TODO` comments that will guide you to where you should make changes to the code). If you need a hint or want to verify your changes, look at the complete version in the `solution` subdirectory.
+
+## Setup
+
+You'll need two terminal windows for this exercise.
+
+1. In all terminals, change to the `exercises/handling-errors/practice` directory.
+2. In one terminal, run `npm install` to install dependencies.
+
+## Part A: Defining Errors for Activities
+
+In this part of the exercise, you will throw an Application Failure that will fail your Activities.
+
+Application Failures are used to communicate application-specific failures in Workflows and Activities. In Workflows, if you throw an `ApplicationFailure`, the Workflow Execution will fail. In Activities, you can either throw an `ApplicationFailure` or another Error to fail the Activity Task. 
```

Comment:
Application Failures ~~are used to~~ communicate application-specific failures in Workflows and Activities. ~~In Workflows, if you throw an~~ An `ApplicationFailure` thrown from a Workflow fails the Workflow Execution ~~will fail~~. To fail an Activity Task ~~In Activities~~, you can either throw an `ApplicationFailure` or another Error ~~to fail the Activity Task~~.

Question. Error capitalized here?